run/sql.mysql/tableCreates.sql

```diff
@@ -102,7 +102,7 @@ create table bmsql_order_line (
   ol_supply_w_id  integer,
   ol_quantity     integer,
   ol_dist_info    char(24),
-  constraint pk_order_line primary key (ol_w_id, ol_d_id, ol_o_id, ol_number)
+  constraint pk_order_line primary key (ol_w_id, ol_d_id, ol_o_id, ol_i_id, ol_number)
```

Comment:
Not sure whether we should add it.

Reply:
it seem always got different `ol_number` value when two record's `ol_w_id, ol_d_id, ol_o_id` have same value from code perspective...https://github.com/lysu/benchmarksql/blob/e5793efbcd378594423e6cfaaea19993144dc87b/src/client/jTPCCTData.java#L424